packages/dev/s2-docs/src/Layout.tsx

```diff
@@ -96,36 +96,15 @@ export function Layout(props: PageProps & {children: ReactElement<any>}) {
             lg: 'none'
           }
         })}>
-        <div
-          className={style({
-            isolation: 'isolate',
-            display: 'flex',
-            flexDirection: 'column',
-            alignItems: 'center',
-            maxWidth: {
-              default: 'full',
-              lg: 1280
-            },
-            marginX: 'auto',
-            marginY: 0,
-            padding: {
-              default: 0,
-              lg: 12
-            },
-            paddingBottom: 0,
-            gap: {
-              default: 0,
-              lg: 12
-            }
-          })}>
```

Comment:
Were these styles intentionally removed?

Reply:
oh whoops, must of messed the rebase for this one, thanks for catching